src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Introduce.scss';
+
+const Introduce = () => {
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+  }
+
+  const handleInput = () => {
+    console.log('input 태그 동작');
+  }
+
+  const handleClick = () => {
+    console.log('button 태그 동작');
+  }
+
+  return(
+    <div className="container">
```

Comment:
```suggestion
    <div className="introduce">
```
scss 사용시에는 가장 부모의 태그에 className에 컴포넌트 이름과 동일한 이름을 넣는 것이
위코드의 컨벤션입니다!